playbooks/group_vars/load-balancer-v2-ci/public.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+# GENERAL #####################################################################
+
+LOAD_BALANCER_SERVER_IP: "{{ COMMON_FLOATING_IP or ansible_host }}"
+cert_manager_server_address: "51.38.226.128"
```

Comment:
This is a public repo, can you move this IP address to the `ansible-secrets` repo?